tkg/client/management_components.go

```diff
@@ -120,7 +141,11 @@ func (c *TkgClient) getTKGPackageConfigValuesFile(managementPackageVersion, kube
 		return "", err
 	}
 
-	valuesFile, err := managementcomponents.GetTKGPackageConfigValuesFileFromUserConfig(managementPackageVersion, userProviderConfigValues, tkgBomConfig)
+	addonsManagerPackageVersion, err := c.getAddonsManagerPackageversion(managementPackageVersion)
```

Comment:
Can we only do this for upgrades ? For fresh install would let addons-manager version be empty since old addons-manager is not expected to be there